packages/manager/src/managers/createSliceMachineManagerClient.ts

```diff
@@ -0,0 +1,23 @@
+// TODO: Fix r19 to allow importing from `r19/client`
+import { createRPCClient, CreateRPCClientArgs, RPCClient } from "r19/client";
+
+// !!! Never import anything other than types from
+// !!! `./createSliceMachineManagerServer` in this file.
```

Comment:
It could, but it would add more complexity. I think it's simpler to keep the types close together in the same file. We would still need some kind of disclaimer to ensure nothing except types are imported in `createSliceMachineManagerClient().ts`.

(For background on why this comment is there: This file is only to be used in browsers. We don't want to accidentally import server code. We use the imported type to provide method types to the browser RPC client, which are derived from the RPC server.)